crates/webauthn/auth/src/webauthn.cairo

```diff
@@ -97,7 +97,7 @@ fn verify_challenge(
 ) -> Result<(), AuthnError> {
     let mut i: usize = 0;
     let mut encoded = Base64UrlFeltEncoder::encode(challenge);
-    let encoded_len: usize = encoded.len();
+    let encoded_len: usize = encoded.len() - 1; // Ignore base64 padding '='
```

Comment:
Will it always be padded?

Reply:
Yeah, the verification uses the felt encoder, this always appends a `=` https://github.com/keep-starknet-strange/alexandria/blob/bbc010b922d9ca6f807ef01b0b255e34331b5eac/src/encoding/src/base64.cairo#L138

Reply:
Ah got it. We should probably include the encoder in our project and modify it to avoid the extra steps